x-pack/plugins/siem/public/management/pages/policy/view/ingest_manager_integration/configure_datasource.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { useLocation } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import { useKibana } from '../../../../../../../../../src/plugins/kibana_react/public';
+import { LinkToApp } from '../../../../../common/components/endpoint/link_to_app';
+import { CustomConfigureDatasourceContent } from '../../../../../../../ingest_manager/public';
+import { getManagementUrl } from '../../../..';
+
+export const ConfigureEndpointDatasource = memo<CustomConfigureDatasourceContent>(
+  ({ from }: { from: string }) => {
+    const { services } = useKibana();
+    const pathname = useLocation().pathname.split('/');
```

Comment:
I don't suggest us using custom logic to parse out the URL. I suggest that this component be given the `datasourceId` as an input prop, and then drive the UI below based on that. It may also address my earlier comment about moving code from Ingest here.